client/src/styles/trumps/icon.fonts.css

```diff
@@ -438,6 +438,10 @@ Generated using nucleoapp.com
   content: "\ea59";
 }
 
+.icon-utilities::before {
+  content: "\ea06";
+}
+
```

Comment:
Is this change still necessary, now that we default to the `icon-applications` icon?

Reply:
Nope, no longer necessary. I'll take that rule out.